### PR TITLE
356 related blog posts

### LINKED ISSUE #356: Related blog posts can be added to work pages
## Description

A list of related blog posts can be added manually to a work page. These are displayed after the main content at the bottom of the page, appearing as they do on the blog index page.

[design file here](https://www.figma.com/file/rLewb9kn4mUt8lLI0wuC7f/GFSC?type=design&node-id=404-862&mode=design&t=x9EwBz2Yk4uYge3r-0)

Screenshot 2023-08-07 at 16 28 19

## Acceptance criteria

- [ ] I can add a list of projects to blog posts in any order
- [ ] If any blog posts exist, a related blog posts section is visible containing blog post previews in reverse chronological order
- [ ] If no blog posts exist, the related blog posts section is not shown

### PR DESCRIPTION
**Pull request by aaaaargZombies**
Fixes #356 

## Description

- a bit of refactoring
  -  `relatedBlogs` partial now just returns a slice
  - I made a `blogTeaser` partial, the intention was to use this everywhere but I needed to change the padding and header levels so
  - I made the `relatedBlogTeaser` partial
-  with related content https://436eb358.gfsc.pages.dev/project/the-towers/
- without related content https://436eb358.gfsc.pages.dev/project/placecal

@geeksforsocialchange/developers

### CODE REVIEW

**Review by r-ferrier — CHANGES_REQUESTED** [review]
This looks functionally great! Just a typo & a slight style difference that need changing and a refactoring suggestion that's optional.

File: themes/gfsc/layouts/partials/relatedBlogTeaser.html [hunk 1]
```diff
@@ -0,0 +1,49 @@
+{{ $page := . }}
+{{ $blogauthors := split $page.Params.Author ", " }}
+<article
+  itemscope
+  itemtype="http://schema.org/BlogPosting"
+  role="article"
+  class="teaser"
+>
+  <div class="teaser__content teaser__content--related">
```
[the remark this answers is not in the document]
Comment (aaaaargZombies) [reply themes/gfsc/layouts/partials/relatedBlogTeaser.html:9]: sure, I was in two minds about whether to do it this way or the what you're suggesting.

**Comment by aaaaargZombies** [comment]
ok I think this is all the issues tidied up. some slight confusion with `.` which is the `this` of hugo but we got there in the end

**Review by r-ferrier — APPROVED** [review]
Nice! Looks good to me!

**Pull request merged**